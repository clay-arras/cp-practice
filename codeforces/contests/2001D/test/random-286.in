1
25
15 5 18 3 4 7 19 12 2 3 24 4 7 11 13 20 9 18 25 9 18 9 1 7 14
