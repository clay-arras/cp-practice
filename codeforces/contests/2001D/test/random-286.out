17
15 5 19 12 2 3 24 4 11 13 20 9 25 18 1 7 14
